static/results.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="utf-8">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src https://raw.githubusercontent.com/;">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; connect-src https://depobs.dev.mozaws.net/;">
```

Comment:
We may not need to specify connect-src since it's all 'self'? We can test this, I guess

Reply:
Lets leave that there for now as it just makes it that bit easier for local testing